`panic: runtime error` when deleting an untracked release

## Helm and Helmsman versions
```
$ helm version
version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}

$ helmsman version

 _          _
| |        | |
| |__   ___| |_ __ ___  ___ _ __ ___   __ _ _ __
| '_ \ / _ \ | '_ ` _ \/ __| '_ ` _ \ / _` | '_ \
| | | |  __/ | | | | | \__ \ | | | | | (_| | | | |
|_| |_|\___|_|_| |_| |_|___/_| |_| |_|\__,_|_| |_| version: v3.4.2
A Helm-Charts-as-Code tool.
```

## Steps to reproduce
1. Create `helmsman.toml` file:
```
context = "pablo"

[settings]
  kubeContext = "minikube"

[namespaces]
  [namespaces.kube-system]
  [namespaces.pablo]

[helmRepos]
  stable = "https://kubernetes-charts.storage.googleapis.com"
  bitnami = "https://charts.bitnami.com/bitnami"

[apps]
  [apps.apache]
    namespace   = "pablo"
    chart       = "bitnami/apache"
    version     = "7.3.16"
    name        = "apache"
    enabled     = true
```

2. Run `helmsman -apply -verbose -f helmsman.toml`
3. Remove the `[apps.apache]` section from `helmsman.toml`.
4. Run `helmsman -apply -verbose -f helmsman.toml` again.
5. the following error happens:

```
[...]
2020-05-29 08:55:36 NOTICE: -------- PLAN starts here --------------
2020-05-29 08:55:36 WARNING: Untracked release [ apache ] found and it will be deleted -- priority: -1000
2020-05-29 08:55:36 NOTICE: -------- PLAN ends here --------------
2020-05-29 08:55:36 INFO: Sorting the commands in the plan based on priorities (order flags) ...
2020-05-29 08:55:36 INFO: Executing plan...
2020-05-29 08:55:36 INFO: Plan applied
2020-05-29 08:55:36 INFO: Cleaning up sensitive and temp files
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x18c8e9c]

goroutine 44 [running]:
github.com/Praqma/helmsman/internal/app.(*plan).exec.func1(0xc00038c340, 0xc0003ea180, 0x1b1e355, 0x4, 0xc0003b8240, 0x4, 0x4, 0xc000394bc0, 0x3a, 0xfffffffffffffce0, ...)
	github.com/Praqma/helmsman/internal/app/plan.go:119 +0x15c
created by github.com/Praqma/helmsman/internal/app.(*plan).exec
	github.com/Praqma/helmsman/internal/app/plan.go:111 +0x4e2
```

I've been able to reproduce the above both on minikube and EKS using different charts.

Facing the same issue.

`2020-06-02 12:57:44 NOTICE: -------- PLAN ends here --------------
2020-06-02 12:57:44 INFO: Executing plan...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xcc3fdc]

goroutine 140 [running]:
github.com/Praqma/helmsman/internal/app.(*plan).exec.func1(0xc0004c4620, 0xc0000824e0, 0xf18156, 0x4, 0xc0000b4780, 0x6, 0x8, 0xc00019e2d0, 0x48, 0xfffffffffffffce0, ...)
	/go/src/github.com/Praqma/helmsman/internal/app/plan.go:119 +0x15c
created by github.com/Praqma/helmsman/internal/app.(*plan).exec
	/go/src/github.com/Praqma/helmsman/internal/app/plan.go:111 +0x4e2
[jenkins@ip-10-30-110-175 app2-perf]$ helm version
version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}`

--- a/internal/app/plan.go
+++ b/internal/app/plan.go
@@ -114,14 +114,14 @@ func (p *plan) exec() {
 					<-sem
 				}()
 				for _, c := range cmd.beforeCommands {
-					execOne(c, cmd.targetRelease.Name)
+					execOne(c, cmd.targetRelease)
 				}
-				execOne(cmd.Command, cmd.targetRelease.Name)
+				execOne(cmd.Command, cmd.targetRelease)
 				if cmd.targetRelease != nil && !flags.dryRun && !flags.destroy {
 					cmd.targetRelease.label()
 				}
 				for _, c := range cmd.afterCommands {
-					execOne(c, cmd.targetRelease.Name)
+					execOne(c, cmd.targetRelease)
 				}
 				// if result.code != 0 {
 				// 	errorMsg := result.errors
@@ -157,7 +157,7 @@ func (p *plan) exec() {
 }
 
 // execOne executes a single ordered command
-func execOne(cmd command, targetRelease string) {
+func execOne(cmd command, targetRelease *release) {
 	log.Notice(cmd.Description)
 	result := cmd.exec()
 
@@ -166,7 +166,12 @@ func execOne(cmd command, targetRelease string) {
 		if !flags.verbose {
 			errorMsg = strings.Split(result.errors, "---")[0]
 		}
-		log.Fatal(fmt.Sprintf("Command for release [%s] returned [ %d ] exit code and error message [ %s ]", targetRelease, result.code, strings.TrimSpace(errorMsg)))
+		if targetRelease != nil {
+			log.Fatal(fmt.Sprintf("Command for release [%s] returned [ %d ] exit code and error message [ %s ]", targetRelease.Name, result.code, strings.TrimSpace(errorMsg)))
+		} else {
+			log.Fatal(fmt.Sprintf("%s returned [ %d ] exit code and error message [ %s ]", cmd.Description, result.code, strings.TrimSpace(errorMsg)))
+		}
+
 	} else {
 		log.Notice(result.output)
 		log.Notice("Finished: " + cmd.Description)

